AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.h

```diff
@@ -199,6 +199,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (MSAnalyticsTransmissionTarget *)transmissionTargetForToken:(NSString *)token;
 
+/**
+ * Set the send time interval for non-critical logs.
+ * Must be between 3 seconds and 86400 seconds.
```

Comment:
```suggestion
 * Must be between 3 seconds and 1 day.
```

Reply:
I prefer to keep both values in seconds since the method takes a number in seconds

```suggestion
 * Must be between 3 seconds and 86400 seconds (1 day).
```